Data/test.lua

```diff
@@ -0,0 +1,9646 @@
+SetSource({
+  SourceLong="Core Rulebook",
+  SourceShort="CR",
+  SourceWeb="http://paizo.com/store/downloads/pathfinder/pathfinderRPG/v5748btpy88yj",
+  SourceDate="2009-08",
+})
+DefineClass({
+  HitDie=12,
+  MaxLevel=20,
+  SkillPointsPerLevel=4,
+  ExClass="Ex-Barbarian",
+  Roles={
+    "Combat",
+    "Skill",
+  },
+  Types={
+    "Base",
+    "PC",
+  },
+  Facts={
+    ClassType="PC",
+    Abb="Brb",
+  },
+  Levels={
+    {
+      Level="1",
+      Abilities={
+        {
+          Category="Class",
+          Nature="AUTOMATIC",
+          Name="Barbarian",
+        },
+      },
+      WeaponBonusProficiencySelections={
+      },
+    },
+  },
+  Name="Barbarian",
```

Comment:
Is this just how your converter works? It's really hard to figure out what class these are since "Name" is so far down the definition.

Reply:
Its because Name is a property that multiple things have, its in a base class. I can move it closer to the top though I think.